src/components/CommitteeTools/components/SelectKeyModal/index.scss

```diff
@@ -134,6 +138,26 @@
         width: 100%;
       }
     }
+    &__button {
+      &.loading {
+        position: relative;
+  
+        &:after {
+          content: "";
+          width: 100%;
+          height: 100%;
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+          opacity: 1;
+          background-position: center;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../../assets/icons/loading.svg");
```

Comment:
@mwogates we have a `<Loading />` component. Please use it instead of rewriting styles. This component also accepts boolean prop `spinner`.